prow/config.gen.yaml

```diff
@@ -2025,6 +2024,310 @@ presubmits:
             memory: 4Gi
 
   maistra/istio:
+  - name: istio-integration-helm-2.4
+    trigger: (?m)^/test( | .* )integration\-helm,?($|\s.*)
+    rerun_command: /test integration-helm
+    skip_report: false
+    max_concurrency: 2
+    always_run: true
+    branches:
+      - ^maistra-2.4$
+    decorate: true
+    path_alias: istio.io/istio
+    spec:
+      containers:
+        - command:
+            - entrypoint
+            - prow/integ-suite-kind.sh
+            - test.integration.helm.kube.presubmit
+          env:
+            - name: GOFLAGS
+              value: -mod=vendor
+            - name: BUILD_WITH_CONTAINER
+              value: "0"
+            - name: INTEGRATION_TEST_FLAGS
+              value: ' --istio.test.retries=1 '
+          # FIXME: Uncomment this when we have our proxy in place
+          # - name: ISTIO_ENVOY_BASE_URL
+          #   value: https://storage.googleapis.com/maistra-prow-testing/proxy
+
+          # FIXME: Using upstream for now, replace with our image
+          # image: "quay.io/maistra-dev/maistra-builder:2.4"
+          image: "gcr.io/istio-testing/build-tools:release-1.16-034c1253b7ce447805ce0e76d39a41ab8e466fea"
```

Comment:
@jwendell Is this workaround still relevant? I took it from 2.3 jobs and changed the tag to 1.16 release but I don't know if this what we still need.